keyboards/ergodox_ez/keymaps/dvorak_42_key/keymap.c

```diff
@@ -5,6 +5,7 @@
 
 
 // to build this keymap
+// source ~/qmk_utils/activate_wsl.sh
```

Comment:
```suggestion
```
This file should already be sourced by your .bashrc. I also plan to remove it anyway, so this instruction will be outdated soon.

Reply:
done, made the change